test/input/sf-light/sf-light.conf

```diff
@@ -34,8 +34,8 @@ beam.agentsim.agents.modalBehaviors.lccm.paramFile = ${beam.inputDirectory}"/lcc
 beam.agentsim.agents.vehicles.bicycles.useBikes=false
 #BeamVehicles Params
 beam.agentsim.agents.vehicles.beamFuelTypesFile = ${beam.inputDirectory}"/beamFuelTypes.csv"
-beam.agentsim.agents.vehicles.beamVehicleTypesFile = ${beam.inputDirectory}"/vehicleTypes.csv"
-beam.agentsim.agents.vehicles.beamVehiclesFile = ${beam.inputDirectory}"/vehicles.csv"
+beam.agentsim.agents.vehicles.beamVehicleTypesFile = ${beam.inputDirectory}"/sample/1k/vehicleTypes.csv.gz"
```

Comment:
Were these left in from other testing, or should these be kept as their normal default?

Reply:
The previous file name was incompatible with household file